### DISCUSSION TITLE
Welcome to FadCam Discussions!

### DISCUSSION THREAD

**Discussion by anonfaded** [anonfaded/FadCam#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Doocter** [comment]
All i say is to add features from a existing app into FadCam

https://play.google.com/store/apps/details?id=me.jagar.xscamera2

**Reply by anonfaded** [reply]
Good suggestion, for sure I'll add features with time. Its my side project and I am right now alone working on it so it can take some time to implement such features. 
Right now I'm adding a bottom navbar and then will start adding new features :)
![Screenshot_20240714-092502.jpg](https://github.com/user-attachments/assets/44d76b4c-65b3-45c3-8019-a8286f6b7153)

Also can you share about how you found this FadCam application? And also the reason for not using that app which link you shared, is due to ads?
